setup.m

```diff
@@ -0,0 +1,23 @@
+% Run this file once to set up your Matlab session for Neuron interaction.
+
+% User setting:
+NeuronInstallationDirectory = 'C:\nrn';
+
+% All dependencies of the generated interface library must be findable.
+% WINDOWS: Put them on the PATH
+dllpath = fullfile(NeuronInstallationDirectory, 'bin');
+syspath = getenv('PATH'); 
+setenv('PATH', [dllpath pathsep syspath]);
+
+% Create definition file for NEURON library.
+HeaderFilePath = "bin/nrnmatlab.h";

```

Comment:
At some point, don't want to regenerate the interface library every new matlab session. Only want to have all paths set up correctly, and have a dedicated function to (re)generate the interface. Probably will want to generate-if-not-exist.

Since we are going for having a dedicated header file, and therefor the definition file should not need manual changes, can have the generateLibraryDefinition and build in a single function.

Reply:
Moved to issue.